examples/gno.land/r/stackdump/home/patch001/patch.gno

```diff
@@ -0,0 +1,10 @@
+package patch
+
+import (
+	index "gno.land/r/stackdump/home"
+)
+
+func Bug() {
+	crossing()
+	index.Set("foo", "hello world")
```

Comment:
Here we make an inter-realm call without having the crossing directive set

What happens: it works

What we expect to happen: Error: missing cross before external crossing()